lib/tasks/manageiq/webpacker.rake

```diff
@@ -1,33 +1,34 @@
-if File.exist? 'manageiq-ui-classic.gemspec'
```

Comment:
Well.. it could.. except with @NickLaMuro 's changes here, we actually need those tasks visible from manageiq :(.

(Previously, we would run webpack:compile from manageiq, which re-ran rake with the ui-classic's env and could call those tasks. But now that we're just using `invoke`.. can't do that :(.)